internal/pkg/describe/status_describe.go

```diff
@@ -216,6 +216,9 @@ func (s *ecsStatusDescriber) ecsServiceAutoscalingAlarms(cluster, service string
 	if err != nil {
 		return nil, fmt.Errorf("retrieve auto scaling alarm names for ECS service %s/%s: %w", cluster, service, err)
 	}
+	if alarmNames == nil {
```

Comment:
nit: like the previous one to catch the empty slice condition
```suggestion
	if len(alarmNames) == 0 {
```